helmfile/values/userCRDs/userCRDs.yaml.gotmpl

```diff
@@ -0,0 +1,29 @@
+{{- if eq .Environment.Name "workload_cluster" }}
```

Comment:
Do we need this check?

Reply:
No it should not be needed any more :+1: